src/frontend/admin/src/layouts/dashboard/nav/account/DashboardLayoutNavAccount.tsx

```diff
@@ -36,13 +42,34 @@ const messages = defineMessages({
   },
 });
 
+type AccountMenuItem = {
+  label: string;
+  clickCallback: () => void;
+};
+
 export function DashboardLayoutNavAccount() {
   const { user } = useAuthContext();
   const ref = useRef<HTMLButtonElement>(null);
   const [navAccountMenuIsOpen, setNavAccountMenuIsOpen] = useState(false);
   const intl = useIntl();
   const theme = useTheme();
 
+  const menuItems: AccountMenuItem[] = [
+    {
+      label: intl.formatMessage(messages.djangoAdmin),
+      clickCallback: () => {
+        if (!process.env.NEXT_PUBLIC_DJANGO_ADMIN_BASE_URL) {
+          return;
+        }
+        window.location.replace(process.env.NEXT_PUBLIC_DJANGO_ADMIN_BASE_URL);
```

Comment:
For a11y purpose a redirection should appear as a `<a href="..."/>`, do you think there is a way to achieve that with MUI?